.github/workflows/deploy-production.yml

```diff
@@ -30,6 +30,10 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0  # Shallow clones should be disabled for a better relevancy of analysis
+      - name: Install Dependencies
+        run: npm install
+      - name: Test and coverage
+        run: npm run test:coverage
```

Comment:
If possible, reuse the test coverage information from the previous job to avoid duplication.

Reply:
![image](https://user-images.githubusercontent.com/70852167/192525273-a4da7fcf-1bd5-4d76-b1ad-e8c2afe3758f.png)
Hello Jo, we have tried that approach however, after spending an entire day trying to figure out how to do it, we came across the sonarcloud docs and the recommended way to generate the coverage report was like this.